setup.py

```diff
@@ -5,21 +5,22 @@ def readme():
         return f.read()
 
 setup(name="hv-uploader",
-      version="1.0",
+      version="1.1",
```

Comment:
Any thoughts on how to consistently version this file? Does the version have to be in this `x.y` format?